d2renderers/d2svg/d2svg.go

```diff
@@ -2018,96 +2021,108 @@ func Render(diagram *d2target.Diagram, opts *RenderOpts) ([]byte, error) {
+		}
 	}
 
 	return out, nil
 }
 
-func singleThemeRulesets(diagramHash string, themeID int64, overrides *d2target.ThemeOverrides) (rulesets string, err error) {
+func singleThemeRulesets(classPrefix string, themeID int64, overrides *d2target.ThemeOverrides) (rulesets string, err error) {
 	out := ""
 	theme := d2themescatalog.Find(themeID)
 	theme.ApplyOverrides(overrides)
 
 	// Global theme colors
 	for _, property := range []string{"fill", "stroke", "background-color", "color"} {
 		out += fmt.Sprintf(`
-		.%s .%s-N1{%s:%s;}
-		.%s .%s-N2{%s:%s;}
-		.%s .%s-N3{%s:%s;}
-		.%s .%s-N4{%s:%s;}
-		.%s .%s-N5{%s:%s;}
-		.%s .%s-N6{%s:%s;}
-		.%s .%s-N7{%s:%s;}
-		.%s .%s-B1{%s:%s;}
-		.%s .%s-B2{%s:%s;}
-		.%s .%s-B3{%s:%s;}
-		.%s .%s-B4{%s:%s;}
-		.%s .%s-B5{%s:%s;}
-		.%s .%s-B6{%s:%s;}
-		.%s .%s-AA2{%s:%s;}
-		.%s .%s-AA4{%s:%s;}
-		.%s .%s-AA5{%s:%s;}
-		.%s .%s-AB4{%s:%s;}
-		.%s .%s-AB5{%s:%s;}`,
-			diagramHash,
+		%s .%s-N1{%s:%s;}
+		%s .%s-N2{%s:%s;}
+		%s .%s-N3{%s:%s;}
+		%s .%s-N4{%s:%s;}
+		%s .%s-N5{%s:%s;}
+		%s .%s-N6{%s:%s;}
+		%s .%s-N7{%s:%s;}
+		%s .%s-B1{%s:%s;}
+		%s .%s-B2{%s:%s;}
+		%s .%s-B3{%s:%s;}
+		%s .%s-B4{%s:%s;}
+		%s .%s-B5{%s:%s;}
+		%s .%s-B6{%s:%s;}
+		%s .%s-AA2{%s:%s;}
+		%s .%s-AA4{%s:%s;}
+		%s .%s-AA5{%s:%s;}
+		%s .%s-AB4{%s:%s;}
+		%s .%s-AB5{%s:%s;}`,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N1,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N2,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N3,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N4,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N5,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N6,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.Neutrals.N7,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B1,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B2,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B3,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B4,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B5,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.B6,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.AA2,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.AA4,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.AA5,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.AB4,
-			diagramHash,
+			classPrefix,
 			property, property, theme.Colors.AB5,
 		)
 	}
 
 	// Appendix
-	out += fmt.Sprintf(".appendix text.text{fill:%s}", theme.Colors.Neutrals.N1)
+	out += fmt.Sprintf("%s .appendix text.text{fill:%s}", classPrefix, theme.Colors.Neutrals.N1)
```

Comment:
i don't see why this changes, or any of the lines below which didn't include the diagramHash